gatsby-browser.js

```diff
@@ -0,0 +1 @@
+import './global.css'
```

Comment:
You can use the 'gatsby-browser.js' file to import any files that you want on every page, such as a global CSS file.